themes/src/main/resources/theme/keycloak.v2/account/src/app/keycloak-service/keycloak.service.ts

```diff
@@ -14,10 +14,10 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-import {KeycloakLoginOptions} from "../../../../../../../../../../adapters/oidc/js";
+import Keycloak, { KeycloakLoginOptions } from "../../../../../../../../../../adapters/oidc/js";
```

Comment:
Yes, the types for the definitions of the instance have changed. Since we're no longer exporting a `KeycloakInstance` or `KeycloakAuthorizationInstance` type. These have been replaced with the `Keycloak` and `KeycloakAuthorization` classes, which can be used directly as a type.

I could alias these types so that they can still be imported by the user and mark them as deprecated. For example:

```ts
/**
 * @deprecated Instead of importing 'KeycloakInstance' you can import 'Keycloak' directly as a type.
 */
export type KeycloakInstance = Keycloak;
```

Reply:
Hmm, that sounds like a good idea to me. Do you think it makes sense?

Reply:
Yeah, I think this makes sense in order to maintain compatibility as much as possible.